cognite_toolkit/_cdf_tk/loaders/_resource_loaders/workflow_loaders.py

```diff
@@ -415,28 +410,21 @@ def check_item(cls, item: dict, filepath: Path, element_no: int | None) -> list[
         return warnings
 
     def retrieve(self, ids: SequenceNotStr[WorkflowVersionId]) -> WorkflowVersionList:
```

Comment:
I realise this is an existing pattern, so let's not change it, but wouldn't it be better to fail on zero ids? It does feel like a latent bug..